services/user.py

```diff
@@ -0,0 +1,58 @@
+from typing import Optional
+
+from django.contrib.auth import get_user_model
+
+from db.models import User
+
+
+def create_user(
+        username: str, password: str,
+        email: Optional[str] = None,
+        first_name: Optional[str] = None,
+        last_name: Optional[str] = None
+) -> User:
+    user = get_user_model().objects.create_user(
+        username=username,
+        password=password,
+    )
+
+    if email is not None:
+        user.email = email
+    if first_name is not None:
+        user.first_name = first_name
+    if last_name is not None:
+        user.last_name = last_name
```

Comment:
You can move this to a separate function. Because this code the same as in update_user